data/campaigns/Delfadors_Memoirs/units/Wose_Shaman.cfg

```diff
@@ -4,15 +4,6 @@
     name=_ "Wose Shaman"
     race=wose
     image=units/wose-shaman.png
-    [standing_anim]
-        start_time=0
-        [frame]
-            image="units/wose-shaman.png"
-            halo="halo/wose-stationary-halo[1~8].png:150"
-        [/frame]
-    [/standing_anim]
```

Comment:
Is the standing_anim deliberately removed?

Reply:
Mainline Woses don't have this animation, and there are no animation sprites to support this tags so I didn't find it appropriate to keep this anim.

Reply:
@stevecotton I added them.